tests/prefetch_related/tests.py

```diff
@@ -2073,3 +2074,197 @@ def test_empty_order(self):
             with self.subTest(book=book):
                 self.assertEqual(len(book.authors_sliced), 1)
                 self.assertIn(book.authors_sliced[0], list(book.authors.all()))
+
+
+class PrefetchRelatedCacheWorksWithInheritance(TestCase):
```

Comment:
```suggestion
class PrefetchRelatedMTICacheTests(TestCase):
```

Reply:
I've applied this suggestion, what does MTI stand for?

Reply:
[Multi-table inheritance](https://docs.djangoproject.com/en/5.2/topics/db/models/#multi-table-inheritance) :+1: